pkg/clusterd/disk.go

```diff
@@ -113,6 +118,39 @@ func DiscoverDevices(executor exec.Executor) ([]*sys.LocalDisk, error) {
 	return disks, nil
 }
 
+func deviceMatchWithFilter(device string, filter, metaDevice string) bool {
+	// We need to allow dm- devices (Kubernetes mountpoints) because dm- devices name are used as
+	// name pattern for meta device
+	var allowDeviceNamePattern = regexp.MustCompile("dm")
```

Comment:
Let's move it to the top of the file to line 36/37 where the other vars are, as there is no need to "generate" it every time the function is called, after that the code so far looks good to me